### DISCUSSION TITLE
openlink의 현재 usercount 구하는 기능

### DISCUSSION THREAD

**Discussion by iwog3927** [storycraft/node-kakao#289] [Q&A]
안녕하세요.

getJoinInfo() 를 통해서 해당 링크의 오픈채팅방의 대부분의 정보를 불러올 수는 있는데 현재 인원에 대한 정보는 알 수 없더라구요.
혹시 현재 인원 정보를 불러오는 방법이 있을까요?

**Comment by storycraft** [comment]
getJoinInfo 메서드에서 제공되지 않는 유저수같은 정보는 오픈링크 웹 api 에서 제공되는 것으로 알고 있습니다.
현재 본 라이브러리에서는 오픈링크 웹 api 레퍼를 제공하고 있지 않습니다.
